sources/editor/Stride.Assets.Presentation/Templates/Core/ProjectExecutable.Android/Resources/values/strings.xml

```diff
@@ -1,3 +1,2 @@
 <resources>
-    <string name="app_name"><#= Properties.PackageGameDisplayName #></string>
```

Comment:
<string name="app_name"><Properties.PackageGameDisplayName/></string> works too, doesn't need to be deleted

Reply:
If it was only for TextView which You deleted then ignore comment.

Reply:
Thanks for the hint, do you have experience with .NET6 and android? would be nice to get this PR moving forward a bit...

Reply:
I want to migrate my Android project from Unity so I've already started looking for the source of the problem with Resources.Designer.cs, but it's harder than i thought. I'll focus on it tomorrow.